onnxruntime/core/providers/openvino/qdq_transformations/qdq_stripping.cc

```diff
@@ -338,7 +338,7 @@ static bool CheckDQRuleSet(const NodeUnit& node_unit,
 }
 
 static bool CheckQFeedsIntoQuantizedOutput(const NodeUnit& node_unit,
-                          const std::unordered_map<std::string, std::string> graph_op_data_type) {
+                                           const std::unordered_map<std::string, std::string> graph_op_data_type) {
```

Comment:
Pass the hash map object as reference to avoid multiple copies
```suggestion
                                           const std::unordered_map<std::string, std::string>& graph_op_data_type) {
```

Reply:
These changes are not in the scope of this PR. These are reflected only for lint spacing changes. Could be taken up in future.